linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1503,7 +1501,11 @@ where
         *info2_b.manager.requested_locking.unwrap()
     );
     let bt_certificate = client_2b
-        .burn(AccountOwner::CHAIN, Amount::from_tokens(1))
+        .transfer_to_account(
```

Comment:
`burn`?

Reply:
It is a little difficult since the `.burn` is pointing to a special but unknown address. But we need it to match it with the `SystemOperation::Transfer`.

So, instead I used a `recipient = Account::burn_address(chain_id)` and go from there.